x-pack/plugins/cases/server/routes/api/cases/comments/find_comments.ts

```diff
@@ -44,10 +49,17 @@ export function initFindCaseCommentsApi({ caseService, router, logger }: RouteDe
     async (context, request, response) => {
       try {
         const client = context.core.savedObjects.client;
-        const query = pipe(
-          FindQueryParamsRt.decode(request.query),
-          fold(throwErrors(Boom.badRequest), identity)
-        );
+        const query = {
```

Comment:
I think is best to leave the schema as it was and just `const subCaseId = subCasesEnabled ? query.subCaseId : undefined` and then relay on this variable for all check afterwards.

Reply:
I throw if the query parameter is defined now.